CHANGELOG.md

```diff
@@ -13,11 +13,25 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### `Parameters`
 
+| Old parameter                    | New parameter       |
+| -------------------------------- | ------------------- |
+| `--minimap2_read_mapping_preset` |                     |
+| `--genome`                       |                     |
+| `--igenomes_ignore`              |                     |
```

Comment:
This is not still in the pipeline?

Reply:
Maybe you copied mine, just add what you changed in nextflow_schema :)

Reply:
Yes, exactly